app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cp, source, destination = command.split(" ")
```

Comment:
Recommendation: Use `split` with whitespace as the default separator. First, check the length of the command list before unpacking to avoid unexpected errors.